runbotics/runbotics-scheduler/src/queue/processor/scheduler.processor.ts

```diff
@@ -102,30 +98,20 @@ export class SchedulerProcessor {
         }
 
         const process = await this.processService.findById(job.data.process.id);
-        this.logger.log(
-            `[Q Process] Starting process "${process.name}" | JobID: `,
-            job.id
-        );
-
-        const bot = await this.checkBotAvailability(
-            process.botCollection,
-            process.system,
-            job
-        ).catch((err) => Promise.reject(new Error(err)));
-
-        this.logger.log(`Starting process ${process.name} on bot ${bot.id}`);
-
-        const processInstanceIdentifier =
-            await this.processSchedulerService.startProcess(job.data, bot);
-
-        await this.botGateway.setBotStatusBusy(bot);
-
-        this.logger.log(
-            `[Q Process] Process "${process.name}" freed the queue | JobID: `,
-            job.id
-        );
-
-        return processInstanceIdentifier;
+        if (!isScheduledProcess(job.data)) {
+            return await this.runProcess(process, job);
+        } else {
+            this.queueService.handleAttendedProcess(process, job.data.input)
+                .then(async () => {
+                    return await this.runProcess(process, job);
+                })
+                .catch((err) => {
+                    this.logger.warn(
+                        `[Q Process] Process "${process.name}" has scheduled with/without input variables for other type of process. ${err.message}`
+                    );
+                    return null;
```

Comment:
you must throw an error here or return rejected promise otherwise you won't trigger right flow